README.md

```diff
@@ -1619,6 +1619,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Ron Waller](https://github.com/RonWaller)
   - [Roshan Sharma](https://github.com/imrosun)
   - [Roshan Singh](https://github.com/roshan2152)
+  - [Roi Sela](https://github.com/roiSela)   
```

Comment:
I think your name must be in line 1617, after Rohit Vuppuluri.
Let us know if you need any help updating that. 🙂

Reply:
I dont really see why, but sure yeah, i updated it now.

Reply:
@Panquesito7 Hey, I did what was asked from me.